gpt4all-bindings/python/gpt4all/gpt4all.py

```diff
@@ -308,12 +310,18 @@ def get_model_from_name(model_name: str) -> pyllmodel.LLModel:
             "ggml-mpt-7b-instruct.bin"
         ]
 
+        REPLIT_MODELS = [
+            "ggml-replit-code-v1-3b-f16.bin"
```

Comment:
are we distributing this one unquantized?

(also where are all the `quantize` bins I don't actually know what's been being used to quantize these models? I guess the MPT ones must have been quantized with my `ggml` fork?)